app/src/main/java/com/hisham/ctintegrationsample/CarTrawlerInjector.kt

```diff
@@ -115,4 +125,15 @@ object CarTrawlerInjector {
         dropOfDateTime.add(GregorianCalendar.DAY_OF_MONTH, rentalPeriodDays)
         return dropOfDateTime
     }
+
+    private fun enableCustomCashTreatment(activity: Activity, palette: PaletteDetails, builder: CartrawlerSDK.Builder) {
+        if (hasValidClientId(activity, palette)) {
+            builder.enableCustomCashTreatment()
+            builder.setUSPDisplayType(USPDisplayType.CHECK_STYLE)
```

Comment:
This new display type is not coupled with the custom cash treatment. In the SDK we can select it in a spinner independent of the cash feature.